bin/flatware

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+#
+# This file was generated by Bundler.
```

Comment:
If this is generated when running bundler, should it be gitignored?

Reply:
No, it's an intentionally added [bundler binstub](https://bundler.io/v2.5/man/bundle-binstubs.1.html)